manifests/nodejs.yml

```diff
@@ -91,87 +91,189 @@ tests/:
             express4: v3.19.0
     waf/:
       test_addresses.py:
-        Test_BodyJson: v2.2.0
+        Test_BodyJson: 
+          '*': v2.2.0
+          nextjs: v4.17.0
         Test_BodyRaw: missing_feature
-        Test_BodyUrlEncoded: v2.2.0
-        Test_BodyXml: v2.2.0
+        Test_BodyUrlEncoded: 
+            '*': v2.2.0
+            nextjs: missing_feature  # request.formData() method not supported yet
+        Test_BodyXml: 
+            '*': v2.2.0
+            nextjs: irrelevant  # Body xml is not converted to JSON in nextjs
         Test_ClientIP: missing_feature
-        Test_Cookies: v2.0.0
+        Test_Cookies:
+          '*': v2.0.0
+          nextjs: v4.17.0
         Test_FullGrpc: missing_feature
-        Test_GraphQL: v4.16.0
-        Test_Headers: v2.0.0
+        Test_GraphQL:
+          '*': v4.16.0
+          nextjs: irrelevant # nextjs is not related with graphql
+        Test_Headers:
+          '*': v2.0.0
+          nextjs: v4.17.0
         Test_Lambda: missing_feature
         Test_Method: missing_feature
-        Test_PathParams: v2.0.0
-        Test_ResponseStatus: v2.0.0
-        Test_UrlQuery: v2.0.0
-        Test_UrlQueryKey: v2.6.0
-        Test_UrlRaw: v2.0.0
+        Test_PathParams:
+            '*': v2.0.0
+            nextjs: missing_feature
+        Test_ResponseStatus:
+          '*': v2.0.0
+          nextjs: v4.17.0
+        Test_UrlQuery:
+          '*': v2.0.0
+          nextjs: v4.17.0
+        Test_UrlQueryKey:
+          '*': v2.6.0
+          nextjs: v4.17.0
+        Test_UrlRaw:
+          '*': v2.0.0
+          nextjs: v4.17.0
         Test_gRPC: missing_feature
       test_blocking.py:
         Test_Blocking: v3.19.0
```

Comment:
Don't we need to add framework info for blocking ?

Reply:
Yes, I think that I just forgot this test because they are just working in nexts...
But all test are related with headers, so we should support because are independant. As a iam removing `nextjs` reference from Headers and other independant test, I'll maintain this as it is now.